src/ae.cpp

```diff
@@ -175,6 +179,15 @@ void aeProcessCmd(aeEventLoop *eventLoop, int fd, void *, int )
             (*cmd.pfn)();
 
             delete cmd.pfn;
+            break;
+        }
+        case AE_ASYNC_OP::PostAsynDBFunction:
+        {   //added to support async api IStorage
+            std::unique_lock<decltype(g_lock)> ulock(g_lock, std::defer_lock);
```

Comment:
Follow the pattern in AE_ASYNC_OP::PostFunction.  We don't acquire and release the lock for each loop iteration because it results in worse performance.